scripts/build_test.py

```diff
@@ -517,7 +512,7 @@ def test_generate_build_tasks_to_build_directory(self):
         """Test generate_build_tasks_to_build_directory queues up a
         corresponding number of build tasks according to the given scenario.
         """
-        EXTENSIONS_DIRNAMES_TO_DIRPATHS = {
+        extensions_dirnames_to_dirpaths = {
```

Comment:
Why is this changed to lower case?

Reply:
Because these are defined inside at local method and variable with capital letters are only allowed at the global leve.